os/drivers/lcd/lcd_dev.c

```diff
@@ -93,9 +96,17 @@ static int lcddev_close(FAR struct file *filep);
 static int lcddev_ioctl(FAR struct file *filep, int cmd, unsigned long arg);
 static ssize_t lcddev_read(FAR struct file *filep, FAR char *buffer, size_t buflen);
 static ssize_t lcddev_write(FAR struct file *filep, FAR const char *buffer, size_t buflen);
+
 /****************************************************************************
  * Private Data
  ****************************************************************************/
+static uint8_t *lcd_kbuffer;
+FAR struct lcd_s *g_lcd_priv;
+FAR const struct lcddev_area_s *g_lcd_area;
+sem_t flushing_sem;
+sem_t ui_wait_sem;
```

Comment:
`ui_wait_sem` is not used. please remove this.

Reply:
done